src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpectatorView.cs

```diff
@@ -118,6 +118,10 @@ public class SpectatorView : MonoBehaviour
         [SerializeField]
         private bool debugLogging = false;
 
+        [Tooltip("Check to hide the developer console every update.")]

```

Comment:
>[Tooltip("Check to hide the developer console every update.")] [](start = 8, length = 62)

I think it might be worth clarifying that this "developer console" is the on-device console, not related to the console pane in Unity